src/sample.cpp

```diff
@@ -184,22 +147,7 @@ void Sample::WriteSample(FileWriter &writer) const
 {
 	assert(this->sample_data != NULL);
 
-	writer.WriteDword('FFIR');
-	writer.WriteDword(this->size - 8);
-	writer.WriteDword('EVAW');
-
-	writer.WriteDword(' tmf');
-	writer.WriteDword(16);
-	writer.WriteWord(1);
-	writer.WriteWord(this->num_channels);
-	writer.WriteDword(this->sample_rate);
-	writer.WriteDword(this->sample_rate * this->num_channels * this->bits_per_sample / 8);
-	writer.WriteWord(this->num_channels * this->bits_per_sample / 8);
-	writer.WriteWord(this->bits_per_sample);
-
-	writer.WriteDword('atad');
-	writer.WriteDword(this->sample_size);
-	writer.WriteRaw(this->sample_data, this->size - RIFF_HEADER_SIZE);
+	writer.WriteRaw(this->sample_data, this->size);
```

Comment:
This breaks the format fix at line 170-172 (original).

Given these variables are not used anymore, they should be removed from sample.hpp.